main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/urfave/cli"
+)
+
+var VERSION = "v0.0.0-dev"
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "Rancher 1.6 to Rancher 2.0 migration-helper"
+	app.Version = VERSION
+	app.Usage = "Please check the options using --help flag"
+	app.Action = func(c *cli.Context) error {
+		StartTool(c)
+		return nil
+	}
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "docker-file",
+			Usage: "Specify absolute path to an alternate Docker compose file (default: docker-compose.yml)",
+		},
+		cli.StringFlag{
+			Name:  "rancher-file",
+			Usage: "Specify absolute path to an alternate Rancher compose file (default: rancher-compose.yml)",
+		},
+		cli.StringFlag{
+			Name:  "kompose-tool-path",
+			Usage: "Specify absolute path to the Kompose tool",
+		},
+	}
+
+	app.Run(os.Args)
+}
+
+func StartTool(c *cli.Context) {
+	if c.GlobalBool("debug") {
+		logrus.SetLevel(logrus.DebugLevel)
+	}
+	textFormatter := &logrus.TextFormatter{
+		FullTimestamp: true,
+	}
+	logrus.SetFormatter(textFormatter)
+
+	logrus.Info("---Starting migration helper tool---")
+
+	t := &ToolArgs{}
+
+	validateFlags(c, t)
+
+	setupConstructsMap(t)
+
+	runParser(t)
+
+	if t.komposeTool != "" {
+		runKompose(t)
+	}
+}
+
+func beforeApp(c *cli.Context) error {
+	if c.GlobalBool("verbose") {
+		logrus.SetLevel(logrus.DebugLevel)
+	}
+	return nil
+}
+
+func validateFlags(c *cli.Context, t *ToolArgs) {
+	if c.IsSet("docker-file") {
```

Comment:
once the default is set on the cli argument, there will be no need for this check :)